mindsdb/utilities/utils.py

```diff
@@ -0,0 +1,34 @@
+import csv
+import re
+import typing
+
+
+def parse_csv_attributes(csv_attributes: str = "") -> typing.Dict[str, str]:
+    """
+    Parse the raw_attributes variable, which uses the CSV format:
+    key=value,another=something_else
+
+    Returns:
+        dict: Parsed key-value pairs as a dictionary.
+    """
+    attributes = {}
+
+    if not csv_attributes:
+        return attributes  # Return empty dictionary if the variable is not set
+
+    try:
+        # Use CSV reader to handle parsing the input
+        reader = csv.reader([csv_attributes])
```

Comment:
Is this the same as `str.split(',')`? if so, maybe that is simpler without the need of the reader

Reply:
Since library csv is also native to python, I don't much the advantage in changing this logic.  It would require to refactor much of this method.

Reply:
CSV is perhaps safer imho as it allows for commas to be used within values due to how CSV allows grouping using parenthesis. All in all probably a more permitting method of storing additional attributes as it'll support simple use cases as well as ones more exotic.